website/docs/language/settings/backends/configuration.mdx

```diff
@@ -71,18 +71,20 @@ If a configuration includes no backend block, Terraform defaults to using the `l
 
 ## Initialization
 
-Whenever a configuration's backend changes, you must run `terraform init` again
+When you change a backend's configuration, you must run `terraform init` again
 to validate and configure the backend before you can perform any plans, applies,
 or state operations.
 
-When changing backends, Terraform will give you the option to migrate
+After you initialize, Terraform creates `.terraform/terraform.tfstate` locally. This file contains the most recent backend configuration, including any authentication parameters you provided to the Terraform CLI. Do not check `.terraform/terraform.tfstate` into Git, as it may contain sensitive credentials for your remote backend.
```

Comment:
Can we limit this just to mention the `.terraform/` directory? Maybe something along the lines of "Terraform stores the most recent backend configuration in the `.terraform/` directory..."? The name of the file is only a historical accident, and it would be great to change eventually just to avoid users discovering and misunderstanding it. 

(the state data is also only stored in `terraform.tfstate` as a default filename using an un-configured local backend, so that is not always the name of the file locally or in remote state storage either)

Reply:
This is a fair point @jbardin - I updated the text to remove the exact file name. Do you like this better?